server/endpoints/Profile.ts

```diff
@@ -9,10 +9,14 @@ export interface NetIdQuery {
   netId: string;
 }
 
+// The type of a query with a token
 export interface ProfileRequest {
   token: string;
 }
 
+/**
+ * [getStudentEmailByToken] returns a student email if the given token is valid
+ */
 export const getStudentEmailByToken: Endpoint<ProfileRequest> = {
   guard: [body("token").notEmpty().isAscii()],
```

Comment:
I hint I wanted to add especially since the method CU Reviews uses to create APIs is unique and kinda weird, if you want to check the parameters taken in, check the `guard` variable and take a look at want is required. So for example here in this `getStudentEmailByToken` function, a `token` variable is required and is being checked to see if it's empty and an ASCII value. This is ensure types are rigid and there isn't much room for error.